README.md

```diff
@@ -79,6 +79,18 @@ Example:
 }
 ```
 
+Field `"header.warning"` allows you to display a message at the top of the dashboard. You can use HTML to configure the field but only a tags and href, target properties are accepted. It's undefined by default.
```

Comment:
I think it will be easier to read if highlight the tag and properties as inline code:

```suggestion
Field `"header.warning"` allows you to display a message at the top of the dashboard. You can use HTML to configure the field but only `<a>` tag and `href`, `target` properties are accepted. It's undefined by default.
```